setup.py

```diff
@@ -77,8 +77,8 @@
         # have to pin it
         'django-filter~=1.0.0',
         'redis~=2.10.6',
-        'asgi-redis~=1.4.3',
-        'channels~=1.1.6',
+        'channels~=2.0',
```

Comment:
Please depend on channels 2.1 as it has some important fixes relevant for observers.